npm-scripts/auto-pr-util.ts

```diff
@@ -0,0 +1,36 @@
+import { Logger } from "logging";
+
+export const getRawIdFromTicketId = (ticketId: string) => {
+    const match = ticketId.match(/(?<=task_)\w+/i);
+    return match ? match[0] : null;
+};
+
+export const getTicketId = (logger: Logger, currentBranchName: string, ticketId?: string) => {
```

Comment:
You don't need the logger as a parameter just for testing. You can create a file-private logger in the global scope and use it directly in `getTicketId`. For testing, all you need to do is something like:
```ts
// In some test
const handler = jest.fn()
addLogHandler(handler)
getTicketId("i am an invalid ticket")
expect(handler).toHaveBeenCalledWith(...)
```
`Logger` takes care of logging to all available `LogHandler`s.

Reply:
Thanks